cypress/integration/team/teams_spec.js

```diff
@@ -42,4 +42,67 @@ describe('Teams Suite', () => {
         // * check the finalUrl
         cy.url().should('include', '/ad-1/channels/town-square');
     });
+
+    it('TS13548 Team or System Admin searches and adds new team member', () => {
+        const user = users['user-1'];
+        const letterCount = 3;
+        const nameStartsWith = user.firstName.slice(0, letterCount);
+        const maxSelectableValues = 20;
+        const teamName = 'Stub team';
+        const teamURL = 'stub-team';
```

Comment:
Please add suffix with random number so that team URL will not collide with existing one (may happen only when running multiple test on local dev).

You may import `getRandomInt` and have it like,
```suggestion
        const teamURL = `stub-team-${getRandomInt(9999).toString()}`;
```

Reply:
Done! Thank you! I Didn't realize there was a create team test already **facepalm**